job descriptions/FE engineer (senior)_Porto.md

```diff
@@ -27,12 +27,13 @@ We're looking for people who improve our team! Our ideal buddy is:
 
 # Nice to have
 * Reactive programming. Strong plus.
-* GraphQL (with Apollo client). Strong plus.
-* HTML5 Canvas for fin-hypergrid library. Strong plus.
-* WebSockets (GraphQL over websockets).
+* Mobx.
+* RxJS (some core modules).
+* HTML5 Canvas. Strong plus.
 * Webpack, PostCSS, cssnext & CSS modules.
 * CSS Flexbox.
-* RxJS (some core modules).
+* GraphQL (with Apollo client). Strong plus.
+* WebSockets (GraphQL over websockets).
```

Comment:
As we're investing in typescript, JSS and Material UI and should be supporting them for some time,  I think we can add them as "nice to have".

Reply:
Add as suggestion in the code review (the first button after the preview tab on the comment editor)